pipeline/beam_tables.py

```diff
@@ -509,6 +536,57 @@ def _write_to_gcs(self, scan_type: str,
         max_writers_per_bundle=0,
         file_naming=_custom_file_naming(suffix='.json.gz')))
 
+  def _write_to_bq_and_gcs(self, scan_type: str,
```

Comment:
maybe refactor out the common pieces for this into write_to_bq and _write_to_gcs individual functions. Or just call both, since calling them serially doesn't stop them from running in parallel on execution.

Reply:
I agree, lets just keep the separate functions and call them each.